javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var array = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
+      array.push(values[i]);
```

Comment:
This function is meant to filter out strings, not collect all numbers. What if the argument also included other data types like` null` or boolean values?